test/test_transforms_v2_refactored.py

```diff
@@ -773,7 +761,10 @@ def test_transform_unknown_size_error(self):
         [torch.Tensor, PIL.Image.Image, datapoints.Image, datapoints.BoundingBox, datapoints.Mask, datapoints.Video],
     )
     def test_noop(self, size, input_type):
-        input = make_input(input_type, spatial_size=self.INPUT_SIZE)
+        input = make_input(
+            input_type,
+            **{"spatial_size" if issubclass(input_type, datapoints.BoundingBox) else "size": self.INPUT_SIZE},
+        )
```

Comment:
I've missed this in my assessment in https://github.com/pytorch/vision/pull/7717#issuecomment-1621236156. The issue here is that we want to test if the resizing is a no-op. Since `resize` acts on the spatial size of a bounding box, we need to make sure to use that here.

Reply:
If we don't want to do that, we could also do 

```python
output = F.resize(input, size=F.get_spatial_size(input), antialias=True)
```

below. Or being more explicit:

```python
output = F.resize(
    input, 
    size=input.spatial_size if isinstance(input, datapoints.BoundingBox) else self.INPUT_SIZE, 
    antialias=True,
)
```

Reply:
I went for 

```python
output = F.resize(input, size=F.get_spatial_size(input), antialias=True)
```